docs/installation.md

```diff
@@ -158,6 +158,14 @@ If using Beacon, first copy the configuration file:
 Then update any config values as needed at `lib/beacon/config/beacon_config.json` 
 and `lib/beacon/config/beacon_cohort.json`.
 
+If using the beacon network, copy the configuration file:
+
+```bash
+./bentoctl.bash init-config beacon-network
+```
+
+and update values at `lib/beacon/config/beacon_network_config.json`.
+
```

Comment:
Can a note about this be added to the v17 migration guide? Similar to here, basically